flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkBenchmarkEnvironment.java

```diff
@@ -272,4 +272,18 @@ private InputGate createInputGate(
 			return gates[0];
 		}
 	}
+
+	private static ShuffleDeploymentDescriptor createLocalSdd(ResultPartitionID resultPartitionID, ResourceID location) {
```

Comment:
Could `ShuffleTestUtils#createSddWithLocalConnection` be used here?

Reply:
we can also use `NettyShuffleDescriptorBuilder` here.